planner/core/plan_to_pb.go

```diff
@@ -416,15 +416,29 @@ func (p *PhysicalIndexScan) ToPB(_ sessionctx.Context, _ kv.StoreType) (*tipb.Ex
 func (p *PhysicalHashJoin) ToPB(ctx sessionctx.Context, storeType kv.StoreType) (*tipb.Executor, error) {
 	sc := ctx.GetSessionVars().StmtCtx
 	client := ctx.GetClient()
-	// todo: mpp na-key toPB.
-	leftJoinKeys := make([]expression.Expression, 0, len(p.LeftJoinKeys))
-	rightJoinKeys := make([]expression.Expression, 0, len(p.RightJoinKeys))
-	for _, leftKey := range p.LeftJoinKeys {
-		leftJoinKeys = append(leftJoinKeys, leftKey)
-	}
-	for _, rightKey := range p.RightJoinKeys {
-		rightJoinKeys = append(rightJoinKeys, rightKey)
+	var leftJoinKeys, rightJoinKeys []expression.Expression
```

Comment:
throw error if both `LeftNAJoinKeys` and `LeftJoinKeys` are not empty?

Reply:
Done.